nominees/scottish-environment-protection-agency-hydrometric-data.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "Scottish Environment Protection Agency Hydrometric Data",
+  "aliases": [
+    "SEPA WISKI API"
+  ],
+  "description": "Provision of Hydrometric Open Data to professional users via API",
+  "website": "https://timeseriesdoc.sepa.org.uk/",
+  "license": [
+    {
+      "spdx": "ODC-By-1.0",
+      "licenseURL": "https://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/"
```

Comment:
Recommend adding OGL-3.0 to the list of valid licenses and then using https://timeseriesdoc.sepa.org.uk/ as the licenseURL (as OGL-3.0 is referenced on this page)

Reply:
I concur with @conradsp. The OGL-3.0 complies with the FSF's [Free Software Definition](https://www.gnu.org/philosophy/free-sw.en.html#fs-definition) and is recognized by Fedora Linux (another DPG) [as a "good" license](https://fedoraproject.org/wiki/Licensing:Main#Good_Licenses) for distributing in their free-as-in-freedom distribution. Adding it as a valid license for the DPG Standard makes sense to me as the best approach.

\*tips fedora to @spotaws\*

(This also could likely be tied into DPGAlliance/DPG-Standard#111 to consolidate new license requests)

Reply:
Fixed in commit 27a55e5